### DISCUSSION TITLE
Version naming convention inconsistency

### DISCUSSION THREAD

**Discussion by BraINstinct0** [valinet/ExplorerPatcher#81] [General]
Up until 22000.194.0.22 there was  a '0' in the third field.
Since 22000.194.23, there is none.
Closing as it is not a bug, but noting in case this was unintended.

**Comment by valinet** [comment]
I have clearly stated that non-bugs should be posted in discussions. Please, let’s follow that rule.

I dropped it because it was useless, especially in that position, as far as I remember. Thus, we can attach a digit at the end for minor revisions of the current release, when needed. I think that was the rationale. Even the actual binaries should be versioned like 22000.258.26.0 at some point, and so on…

**Reply by BraINstinct0** [reply]
Sorry for the nuisance, posting it on the issues wasn't intended but I'll follow the rules from now.

I agree, the Windows Version(two fields) - major version - minor version will work the best.
